app/views/kaminari/ra-twitter-bootstrap/without_count/_paginator.html.haml

```diff
@@ -0,0 +1,4 @@
+= paginator.render do
+  %ul.pagination
+    = prev_page_tag if !current_page.first?
+    = next_page_tag # if !current_page.last? && !current_page.out_of_range?
```

Comment:
This commented-out code should be removed.

Reply:
Thanks, will do.